linera-core/src/client/mod.rs

```diff
@@ -2835,8 +2848,8 @@ where
                 return Ok((certificates, None));
             }
             match self.execute_block(vec![]).await {
-                Ok(ExecuteBlockOutcome::Executed(certificate))
-                | Ok(ExecuteBlockOutcome::Conflict(certificate)) => certificates.push(certificate),
+                Ok(ExecuteBlockOutcome::Executed(certificate)) => certificates.push(certificate),
+                Ok(ExecuteBlockOutcome::Conflict(certificate)) => certificates.push(certificate),
```

Comment:
Doesn't Clippy want you to use `|` here to deduplicate the match arms?

Reply:
Ah, I can do that now. There was an intermediate state where `Conflict` didn't contain the `ConfirmedBlockCertificate` yet. 

But no, `clippy` doesn't complain here.

Reply:
Done https://github.com/linera-io/linera-protocol/pull/2832/commits/e2f4558e190047550dd9fdaf60b7162d9e7a5bc6.